saito-js/lib/block.ts

```diff
@@ -29,31 +27,32 @@ export default class Block extends WasmWrapper<WasmBlock> {
         hash: this.hash,
         type: JSON.stringify(this.block_type),
         previous_block_hash: this.previousBlockHash,
-        transactions: this.transactions.map((tx) => tx.toJson()),
-        cv: this.instance.cv,
+        txs: this.instance.transactions,
```

Comment:
is this generating a json automatically now ?

Reply:
Yes, the transactions array is still available in the block object in the explorer

Reply:
why though? it's not in json format. right? also in the latest code transactions list is totally removed